support.md

```diff
@@ -69,15 +67,15 @@ In the description of your issue, be sure to also include:
     `systemctl status brickman.service -l`.
   * If a screenshot of the EV3 would be helpful, run `fbgrab <some-name>.png`
     on the EV3 and include the picture in your comments.
-<p />
```

Comment:
Is there not a matching `<p>` to delete? Or is this another case of "WTH<sup>*</sup> HTMLProofer, I thought you were suppose to *validate* things".

<sub>* "what the heck" is my preferred exclamation.</sup>